scripts/fix_fork_component_root.py

```diff
@@ -0,0 +1,35 @@
+import logging
+import sys
+
+from django.db import transaction
+
+from website.app import setup_django
+setup_django()
+from osf.models import AbstractNode
+from scripts import utils as script_utils
+
+
+logger = logging.getLogger(__name__)
+
+def main():
```

Comment:
This script looks like it could just be made into an automatic_migration rather than requiring someone to run it manually. It should only have to be done once when this change is in. I don't expect it would need a reverse because there's no good reason to have the wrong root. @sloria do you agree, or do you think this would be too much overhead for day-to-day developers?

Reply:
An example of us doing this was in https://github.com/CenterForOpenScience/osf.io/blob/develop/osf/migrations/0055_update_metaschema_active.py

Reply:
I'm not sure we actually need this. We already have `scripts.fix_root`. I can just run that after this fix is deployed.

Reply:
Okay, we'll take this out and make a note in the deployment.